app/src/main/java/com/example/keybindhelper/MainActivity.kt

```diff
@@ -22,17 +22,15 @@ import com.google.android.material.navigation.NavigationView
 
 
 class MainActivity : AppCompatActivity() {
-
-
     private lateinit var appBarConfiguration: AppBarConfiguration
     private lateinit var binding: ActivityMainBinding
 
-    val allActionMenuIds= setOf(R.id.action_add,R.id.action_delete_all_groups,R.id.action_show_hide_keybinds)
+    private val allActionMenuIds= setOf(R.id.action_add,R.id.action_delete_all_groups,R.id.action_show_hide_keybinds)
     val keybindsFragmentActionMenuIds= setOf(R.id.action_add,R.id.action_delete_all_groups,R.id.action_show_hide_keybinds)
     val projectsFragmentActionMenuIds= setOf(R.id.action_add)
     val shareFragmentActionMenuIds=setOf<Int>()
 
-    var Menu:Menu?=null;
+    var menu:Menu?=null;
```

Comment:
variable names should be camel case
https://kotlinlang.org/docs/coding-conventions.html#function-names

Reply:
Good.